pom.xml

```diff
@@ -334,6 +307,59 @@
   </build>
 
   <profiles>
+    <profile>
+      <id>java9_and_up</id>
+      <activation>
+        <jdk>[9,]</jdk>
+      </activation>
+
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-compiler-plugin</artifactId>
+            <version>3.10.1</version>
+            <configuration>
+              <!-- For Java 8, the plugin is overridden below, without errorprone -->
+              <release>11</release>
+              <showDeprecation>true</showDeprecation>
+              <showWarnings>true</showWarnings>
+              <annotationProcessorPaths>
+                <path>
+                  <groupId>com.google.errorprone</groupId>
+                  <artifactId>error_prone_core</artifactId>
+                  <version>2.11.0</version>
+                </path>
+                <path>
+                  <groupId>com.uber.nullaway</groupId>
+                  <artifactId>nullaway</artifactId>
+                  <version>0.7.8</version>
+                </path>
+              </annotationProcessorPaths>
+              <fork>true</fork>
+              <compilerArgs>
+                <arg>-XDcompilePolicy=simple</arg>
+                <arg>-Xplugin:ErrorProne -Xep:NullAway:ERROR -XepOpt:NullAway:AnnotatedPackages=com.google.cloud.tools -XepOpt:NullAway:KnownInitializers=com.google.gson.Gson.fromJson -XepOpt:NullAway:ExcludedFieldAnnotations=org.mockito.Mock,org.junit.runners.Parameterized -XepOpt:NullAway:ExcludedClasses=${errorprone_exclusions}</arg>
+                <!-- Errorprone requires opening up some JDK modules.
+                     See https://errorprone.info/docs/installation#maven for details
+                -->
+                <arg>-J--add-exports=jdk.compiler/com.sun.tools.javac.api=ALL-UNNAMED</arg>
```

Comment:
Ah looks like we run into the following issues if we don't include these options:
```
 com.sun.jna.Library is not found, referenced from net.bytebuddy.dynamic.loading.ClassInjector$UsingJna$Dispatcher$Jvm
  com.sun.jna.LastErrorException is not found, referenced from net.bytebuddy.dynamic.loading.ClassInjector$UsingJna$Dispatcher$Jvm
  com.sun.jna.Platform is not found, referenced from net.bytebuddy.dynamic.loading.ClassInjector$UsingJna$Dispatcher$CreationAction
  com.sun.jna.Native is not found, referenced from net.bytebuddy.dynamic.loading.ClassInjector$UsingJna$Dispatcher$CreationAction
  com.sun.jna.JNIEnv is not found, referenced from net.bytebuddy.dynamic.loading.ClassInjector$UsingJna$Dispatcher$Enabled
  com.sun.jna.FunctionMapper is not found, referenced from net.bytebuddy.dynamic.loading.ClassInjector$UsingJna$Dispatcher$Windows32BitFunctionMapper
  com.sun.jna.Platform.isWindows is not found, referenced from net.bytebuddy.dynamic.loading.ClassInjector$UsingJna$Dispatcher$CreationAction

```